core/dvector.h

```diff
@@ -232,17 +232,19 @@ class DVector {
 	}
 
 
-	void remove(int p_index) {
+	T remove(int p_index) {
 
+		T ret = get(p_index);
```

Comment:
I made the mistake of adding the get here, has to be after `ERR_FAIL_INDEX_V`, then lets see what happens.